vhdbuilder/packer/components.json

```
@@ -368,17 +369,17 @@
       "downloadLocation": "/opt/cni/downloads",
       "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
       "versions": [
-        "1.4.51",
-        "1.5.15"
+        "1.4.43.1",
+        "1.5.11",
       ]
     },
     {
       "fileName": "azure-vnet-cni-swift-linux-amd64-v*",
       "downloadLocation": "/opt/cni/downloads",
       "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
       "versions": [
-        "1.4.51",
-        "1.5.15"
+        "1.5.11",
+        "1.4.43.1",
```

```suggestion
        "1.4.43.1"
```
dangling `,` breaks the JSON